File: design-patterns/go/chain_of_responsibility/interview/ceo_interview.go
// ceo_interview.go

package main

import "fmt"

type CeoInterview struct {
	nextInterview Interview
}

func NewCeoInterview(nextInterview Interview) (ceoInterview *CeoInterview) {
	ceoInterview = &CeoInterview{}
	ceoInterview.nextInterview = nextInterview
	return
}

func (ceoInterview *CeoInterview) Execute() {
	// Ask all questions for ceo interview
	// Perform any other action required for ceo interview
	fmt.Println("Ask CEO interview questions")

	// Execute the next interview set while creating new struct
	if ceoInterview.nextInterview != nil {
		ceoInterview.nextInterview.Execute()
	}
}

File: design-patterns/php/chainofresponsibility/interview/CeoInterview.php
<?php
// CeoInterview.php

namespace BigBoxCode\DesignPattern\ChainOfResponsibility\Interview;


class CeoInterview extends Interview {
    public function __construct(?Interview $nextInterview) {
        parent::__construct($nextInterview);
    }

    public function execute(): void {
        // Ask all questions for ceo interview
        // Perform any other action required for ceo interview
        echo "Ask CEO interview questions\n";

        // Execute the next interview set while creating new struct
        if ($this->nextInterview) {
            $this->nextInterview->execute();
        }
    }

}